CMakeLists.txt

```diff
@@ -18,10 +18,14 @@ ExternalProject_add(build-clang-format
   USES_TERMINAL_DOWNLOAD 1
   USES_TERMINAL_CONFIGURE 1
   USES_TERMINAL_BUILD 1
-  CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLLVM_ENABLE_ZLIB=OFF -DLLVM_ENABLE_ZSTD=OFF -DLLVM_ENABLE_PROJECTS=clang
-  BUILD_COMMAND ${CMAKE_COMMAND} --build . --target clang-format
+  CMAKE_ARGS -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DLLVM_ENABLE_ZLIB=OFF -DLLVM_ENABLE_ZSTD=OFF -DLLVM_ENABLE_PROJECTS=clang -DLLVM_TARGETS_TO_BUILD=
```

Comment:
Can you give some insight into the additional `-DLLVM_TARGETS_TO_BUILD=`? What does it do?

Reply:
This is list of CPU targets the clang we are building should be able to generate binaries for, e.g. x86, arm, etc
By default this is all supported targets, setting it to empty means it will only use the host architecture, so there are (slightly) fewer files to compile.